src/domains/user/accounts.routes.integration.spec.js

```diff
@@ -7,21 +7,146 @@ describeDbTestSuite('accountsRouter', () => {
   const app = appFactory.createExpressApp();
   const accountsRouter = getAccountsRouter();
   app.use(accountsRouter);
-  describe('create', () => {
+  describe('Test user authentication', () => {
     test('it should create an account successfully', async () => {
       const createAccountBody = {
         email: 'email@example.com',
         password: '0PasSwoRd12',
       };
 
+      const response = await request(app)
```

Comment:
- Make this bit reusable
- Have a function that return the `request(app).post('/signup').set('content')...`
e.g
```
const api = body => 
  request(app)
    .post('/signup')
    .set('content-type', '')
    .send(body)
```
From the test case
```
await api(createAccountBody).then(res => {
  return expect(res.body.message).toEqual('Account created')
})

Reply:
Why do you use await and .then?